src/screens/LinkContextScreen/index.js

```diff
@@ -175,7 +175,7 @@ const styles = StyleSheet.create({
     alignSelf: 'center'
   },
   bottomAnchorTextContainer: {
-    backgroundColor: COLORS.bondi_blue,
+    backgroundColor: COLORS.holyTosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.bondi_blue` to `COLORS.holyTosca`. Ensure that this change aligns with the overall theme and design guidelines of your application. If `COLORS.holyTosca` is not defined in your theme file, this will cause an error.
```diff
-    backgroundColor: COLORS.bondi_blue,
+    backgroundColor: COLORS.holyTosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->